src/Microsoft.AspNetCore.JsonPatch/project.json

```diff
@@ -21,23 +21,14 @@
     ]
   },
   "dependencies": {
-    "Newtonsoft.Json": "9.0.1"
+    "Microsoft.Extensions.ClosedGenericMatcher.Sources": {
+        "type":"build",
+        "version":"1.1.0-*"
+    },
+    "Newtonsoft.Json": "9.0.1",
+    "System.Reflection.TypeExtensions": "4.1.0-*"
   },
   "frameworks": {
-    "netstandard1.1": {
-      "dependencies": {
-        "Microsoft.CSharp": "4.0.1-*",
-        "System.Collections.Concurrent": "4.0.12-*",
-        "System.ComponentModel.TypeConverter": "4.1.0-*",
-        "System.Diagnostics.Debug": "4.0.11-*",
-        "System.Globalization": "4.0.11-*",
-        "System.Linq": "4.1.0-*",
-        "System.Reflection.Extensions": "4.0.1-*",
-        "System.Resources.ResourceManager": "4.0.1-*",
-        "System.Runtime.Extensions": "4.1.0-*",
-        "System.Runtime.Serialization.Primitives": "4.1.1-*",
-        "System.Text.Encoding.Extensions": "4.0.11-*"
-      }
-    }
+    "netstandard1.3": { }
```

Comment:
This is because of the following line: https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.ClosedGenericMatcher.Sources/ClosedGenericMatcher.cs#L68

And the package that has it is in 1.3 http://packagesearch.azurewebsites.net/?q=GetInterfaces